web/src/features/ngos/components/CreateNGODialog.tsx

```diff
@@ -0,0 +1,154 @@
+import { Button } from '@/components/ui/button';
+import { Dialog, DialogClose, DialogContent, DialogFooter, DialogTitle } from '@/components/ui/dialog';
+import { Form, FormField, FormItem, FormLabel, FormMessage } from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { toast } from '@/components/ui/use-toast';
+import { InformationCircleIcon } from '@heroicons/react/24/outline';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useCreateNgo } from '../hooks/ngos-queriess';
+import { newNgoSchema, NgoCreationFormData } from '../models/NGO';
+
+export interface CreateNGODialogProps {
+  open: boolean;
+  onOpenChange: (open: any) => void;
+}
+
+function CreateNGODialog({ open, onOpenChange }: CreateNGODialogProps) {
+  const { t } = useTranslation('translation', { keyPrefix: 'observers.addObserver' });
+  const { createNgoMutation } = useCreateNgo();
+
+  const form = useForm<NgoCreationFormData>({
+    resolver: zodResolver(newNgoSchema),
+  });
+
+  function onSubmit(values: NgoCreationFormData) {
+    createNgoMutation.mutate({
+      values,
+      onMutationSuccess: () => {
+        form.reset({});
+        onOpenChange(false);
+        toast({
+          title: 'Success',
+          description: 'New organization created',
+        });
+      },
+    });
+  }
+
+  return (
+    <Dialog open={open} onOpenChange={onOpenChange} modal={true}>
+      <DialogContent
+        className='min-w-[650px] min-h-[350px]'
+        onInteractOutside={(e) => {
+          e.preventDefault();
+        }}
+        onEscapeKeyDown={(e) => {
+          e.preventDefault();
+        }}>
+        <DialogTitle className='mb-3.5'>Add organization</DialogTitle>
+        <div className='flex flex-col gap-3'>
+          <Form {...form}>
+            <form onSubmit={form.handleSubmit(onSubmit)} className='space-y-4'>
+              <FormField
+                control={form.control}
+                name='name'
+                render={({ field, fieldState }) => (
+                  <FormItem>
+                    <FormLabel>Name</FormLabel>
+                    <Input placeholder='Name' {...field} {...fieldState} />
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+
+              <div className='inline-flex text-slate-700'>
+                <div>
+                  <InformationCircleIcon width={24} height={24} />
+                </div>
+                <div className='ml-2 text-sm'>
```

Comment:
let's simplify this please. I should be able to create a ngo even if I do not know the contact person. The ngo admin can be added later. @avasi-code4ro please update the design as well